File: Sample/Instabug Demo/Support/UIViewController+DefaultAlertController.m
//
//  UIViewController+DefaultAlertController.m
//  Instabug Demo
//
//  Copyright © 2016 Instabug. All rights reserved.
//

#import "UIViewController+DefaultAlertController.h"

@implementation UIViewController (DefaultAlertController)

- (void)ibg_showAlertControllerWithTitle:(NSString *)title
                                 message:(NSString *)message
                                 actions:(NSArray<UIAlertAction *> *)actions {
    UIAlertController *alertController = [UIAlertController alertControllerWithTitle:title
                                                                             message:message
                                                                      preferredStyle:UIAlertControllerStyleActionSheet];
    
    for (UIAlertAction *action in actions) {
        [alertController addAction:action];
    }
    
    UIAlertAction *cancelAction = [UIAlertAction actionWithTitle:@"Cancel"
                                                           style:UIAlertActionStyleCancel
                                                         handler:nil];
    
    [alertController addAction:cancelAction];
    
    [self presentViewController:alertController animated:YES completion:nil];
}

@end

File: Sample/Instabug Demo/Support/UIViewController+DefaultAlertController.swift
//  UIViewController+DefaultAlertController.m
//  Instabug Demo
//
//  Copyright © 2016 Instabug. All rights reserved.
//

import UIKit

extension UIViewController {
    func ibg_showAlert(withTitle title: String?, message: String?, actions: [UIAlertAction]?) {
        let alertController = UIAlertController(title: title, message: message, preferredStyle: .actionSheet)
        for action: UIAlertAction? in actions ?? [UIAlertAction?]() {
            if let anAction = action {
                alertController.addAction(anAction)
            }
        }
        let cancelAction = UIAlertAction(title: "Cancel", style: .cancel, handler: nil)
        alertController.addAction(cancelAction)
        present(alertController, animated: true)
    }
}
